backbone/src/main/java/org/researchstack/backbone/model/User.java

```diff
@@ -68,9 +68,9 @@ public void setDataSharingScope(DataSharingScope dataSharingScope) {
      * ALL   - The user has consented to sharing their de-identified data for current and future research, which may or may not involve the same institutions or investigators.
      */
     public enum DataSharingScope {
-        NONE("none"),
-        STUDY("study"),
-        ALL("all");
+        NONE("no_sharing"),
```

Comment:
Nice. Want to replace literals with this enum and also fix this in your PR? https://sagebionetworks.jira.com/browse/MA-48